$NetBSD: patch-aa,v 1.13 2011/05/12 12:15:41 dmcmahill Exp $

### Fix brokenness where the guile standalone program immediately crashes on startup.
### See http://lists.gnu.org/archive/html/guile-user/2009-12/msg00019.html
### for details.  This was observed on NetBSD-5.1/alpha

--- libguile/eval.c.orig	2010-12-13 17:24:39.000000000 +0000
+++ libguile/eval.c
@@ -3099,7 +3099,7 @@ scm_t_option scm_debug_opts[] = {
   { SCM_OPTION_INTEGER, "depth", 20, "Maximal length of printed backtrace." },
   { SCM_OPTION_BOOLEAN, "backtrace", 0, "Show backtrace on error." },
   { SCM_OPTION_BOOLEAN, "debug", 0, "Use the debugging evaluator." },
-  { SCM_OPTION_INTEGER, "stack", 20000, "Stack size limit (measured in words; 0 = no check)." },
+  { SCM_OPTION_INTEGER, "stack", 60000, "Stack size limit (measured in words; 0 = no check)." },
   { SCM_OPTION_SCM, "show-file-name", (unsigned long)SCM_BOOL_T, "Show file names and line numbers in backtraces when not `#f'.  A value of `base' displays only base names, while `#t' displays full names."},
   { SCM_OPTION_BOOLEAN, "warn-deprecated", 0, "Warn when deprecated features are used." }
 };
